feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.iteneum.repair.data
```

Comment:
The viewModel is going to be created in the next spring so I think you don't need this class

Reply:
It has been changed :)